src/main/deltachat/chatlist.ts

```diff
@@ -96,7 +111,67 @@ export default class DCChatList extends SplitOut {
     return chats
   }
 
-  async getChatListSummary(list: ChatList, i: number) {
+  async getChatListItemsByEntries(entries: [number, number][]) {
+    const label = '[BENCH] getChatListItemsByEntries'
```

Comment:
I think we can uncomment the benchmarking?

Reply:
yes probably